### PR TITLE
Add 2024 BoFs and Poster Presentations blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**

### CODE REVIEW

**Review by bhansa — APPROVED** [review]

**Review by anistark — APPROVED** [review]

File: content/blog/2024/bofs-and-poster-presentations.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Title: Share, Learn, and Connect: Introducing BoF Sessions and Poster Presentations at PyCon India 2024
```
Comment (sayanchowdhury) [inline content/blog/2024/bofs-and-poster-presentations.md:1]:
```suggestion
Title: Share, Learn, and Connect: Introducing BoFs Sessions and Poster Presentations at PyCon India 2024
```

File: content/blog/2024/bofs-and-poster-presentations.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Birds of a Feather (BoF) Sessions
+
+Birds of a Feather sessions, or BoFs, are specially designed to be informal, agenda-free gatherings where participants with similar interests can openly share ideas, challenges, and solutions. 
+These sessions strike a unique balance between seriousness and fun, providing the ideal environment for deep diving into specific topics while fostering robust community engagement and learning. Whether you’re looking to explore new trends, solve complex problems, or simply share experiences with like-minded individuals, BoFs at PyCon India offer a perfect platform to enhance your conference experience.
+
+#### Session Format:
+
+- BoFs will take the form of relaxed panel discussions with active audience participation.
+- The session proposer will serve as the moderator, guiding the discussion to ensure it remains focused and engaging, and encouraging participation from all attendees.
```
Comment (sayanchowdhury) [inline content/blog/2024/bofs-and-poster-presentations.md:18]: Can we add the the discussion will follow `Chatham House` rules and link to `https://www.chathamhouse.org/about-us/chatham-house-rule`

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: content/blog/2024/bofs-and-poster-presentations.md (hunk 1 above)
Comment (palnabarun) [reply content/blog/2024/bofs-and-poster-presentations.md:1]: Why "BoFs" ?

Comment (sayanchowdhury) [reply content/blog/2024/bofs-and-poster-presentations.md:1]: Ah! I see it's followed by Sessions. Skip this

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**